### PR TITLE
docs(config options): rewrite `forkProcessing`

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Rewrite docs for `forkProcessing`
- Monospace word in `description` for the config option

## Context

Drive-by PR, not closing any issue.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/configuration-options.md [hunk 1]
```diff
@@ -994,17 +994,37 @@ If this option is enabled, reviewers will need to create a new PR if additional
...
+### Process a fork in `autodiscover` mode`
+
+If you want Renovate to run on a forked repository when in `autodiscover` mode then:
+
+- Add `"forkProcessing": "enabled"` to your repository config,
+- Or run the CLI command with `--fork-processing=enabled`
+
+### Process a fork in other modes
+
+If you're running Renovate in some other mode, for example when giving a list of repositories to Renovate, but want to skip forked repositories: you must configure `"forkProcessing": "disabled"` in your global config.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:1009]:
```suggestion
If you're running Renovate in some other mode, for example when giving a list of repositories to Renovate, but want to skip forked repositories: set `"forkProcessing": "disabled"` in your _global_ config.
```

File: docs/usage/configuration-options.md [hunk 2]
```diff
@@ -994,17 +994,37 @@ If this option is enabled, reviewers will need to create a new PR if additional
...
+### Process a fork in `autodiscover` mode`
+
+If you want Renovate to run on a forked repository when in `autodiscover` mode then:
+
+- Add `"forkProcessing": "enabled"` to your repository config,
+- Or run the CLI command with `--fork-processing=enabled`
+
+### Process a fork in other modes
+
+If you're running Renovate in some other mode, for example when giving a list of repositories to Renovate, but want to skip forked repositories: set `"forkProcessing": "disabled"` in your _global_ config.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:1009]: I'm out of ideas on how to rewrite this to be shorter or spread over two sentences. Can you suggest a better wording?

**Comment by HonkingGoose** [comment]
The linter passes, but the `build` test shouts at me:

```
FAIL test/website-docs.spec.ts (5.683 s, 1271 MB heap size)
  ● website-docs › has headers for every required sub-option

    expect(received).toEqual(expected) // deep equality

    - Expected  - 0
    + Received  + 4

    @@ -1,6 +1,10 @@
      Array [
    +   "Allowed filenames",
    +   "Process a fork in `autodiscover` mode`",
    +   "Process a fork in other modes",
    +   "When using the hosted GitHub Mend Renovate app",
        "abortIgnoreStatusCodes",
        "abortOnError",
        "allowedVersions",
        "artifactAuth",
        "authType",

      74 |
      75 |   it('has headers for every required sub-option', () => {
    > 76 |     expect(headers3).toEqual(expectedOptions3);
         |                      ^
      77 |   });
      78 |
      79 |   // Checking relatedOptions field in options

      at Object.<anonymous> (test/website-docs.spec.ts:76:22)


Test Suites: 1 failed, 598 passed, 599 total
```

What do you want me to do?

1. Remove headings and use **bold text**, or
2. Let somebody else adjust the test.

**Comment by rarkins** [comment]
(1) use bold

File: docs/usage/configuration-options.md [hunk 3]
```diff
@@ -994,17 +994,37 @@ If this option is enabled, reviewers will need to create a new PR if additional
...
+It even skips a forked repository that has a Renovate configuration file, because Renovate doesn't know if that file was added by the forked repository.
 
-<!-- prettier-ignore -->
-!!! note
-    Only the `onboardingConfigFileName` (which defaults to `renovate.json`) is supported for `forkProcessing`. You cannot use other filenames because Renovate will use the platform API to check only for the default filename.
+**Process a fork in `autodiscover` mode`**
+
+If you want Renovate to run on a forked repository when in `autodiscover` mode then:
+
+- Add `"forkProcessing": "enabled"` to your repository config,
```
Comment (rarkins) [inline docs/usage/configuration-options.md:1004]:
```suggestion
- Ensure a `renovate.json` config exists with `"forkProcessing": "enabled"` in your repository,
```

**Review by rarkins — APPROVED** [review]

**Pull request merged**